Blockzilla/AppDelegate.swift

```diff
@@ -23,6 +23,11 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
         setupErrorTracking()
         setupTelemetry()
         TPStatsBlocklistChecker.shared.startup()
+        
+        // Count number of app launches for requesting a review
+        let currentLaunchCount = UserDefaults.standard.integer(forKey: "launchCount")
+        UserDefaults.standard.set(currentLaunchCount + 1, forKey:"launchCount")
+        UserDefaults.standard.synchronize()
```

Comment:
It is good to consider this, but remove this line. `synchronize()` happens automatically at intervals, and calling directly might be used for system-critical info, and this is ok if in some rare circumstance the count didn't increase. During the app startup process, resource consumption (CPU load, disk and memory I/O, etc.) is at the highest in the app lifetime (for normal usage), so it is best to perform as little work as possible.